### PR TITLE
Tensorflow, Py3.11 and PySide 6

### PR DESCRIPTION
**Pull request by mzieg**
This was created as part of #303 to investigate an issue importing Tensorflow in plugins from Enlighten.exe after building with pyinstaller, but has morphed in unexpected directions :-/

- Python 3.10 -> 3.11
- PySide2 -> 6
- PyInstaller 4 -> 5
- numpy -> 1.26
- Tensorflow <- 2.13

Related pulls:
- https://github.com/WasatchPhotonics/SPyC_Writer/pull/5
- https://github.com/WasatchPhotonics/Wasatch.PY/pull/91

### CODE REVIEW

**Comment by mzieg** [comment]
Reminder to look at this while we're at it: https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community

**Comment by mzieg** [comment]
> Reminder to look at this while we're at it: https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community

This works brilliantly.

File: README_CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
...
 
+- 2023-09-30 4.0.20
+    - updated dependency versions to resolve tensorflow import issue
+        - Python 3.10 -> 3.11
+        - PySide 2 -> 6
+        - pyinstaller 4 -> 5
+        - numpy -> 1.26
+        - tensorflow <- 2.13
+    - let bootstrap pass compression to InnoSetup (default lzma/max)
 - 2023-09-29 4.0.19-RC
```
Comment (samiebee43) [inline README_CHANGELOG.md:13]: RC suffix doesn't need to be in changelog.

**Review by samiebee43 — APPROVED** [review]
Exciting stuff !

**Comment by mzieg** [comment]
final commit resolved #305

**Pull request merged**